openfl/interface/interactive_api/experiment.py

```diff
@@ -62,6 +63,22 @@ def __init__(
         self.logger = getLogger(__name__)
         setup_logging()
 
+        self._initialize_plan()
```

Comment:
Why do we need an initialized plan at an Experiment creation time?

Reply:
The main reason is so that user know what the plan parameters are ahead of time (viewable with fx.get_plan) and can then override them at runtime.